examples/src/main/java/com/mapbox/navigation/examples/core/DebugMapboxNavigationKt.kt

```diff
@@ -65,6 +65,10 @@ import kotlinx.coroutines.channels.Channel
 import okhttp3.Cache
 import timber.log.Timber
 
+/**
+ * This activity shows debug points for raw(red) and
+ * enhanced(blue) location updates.
```

Comment:
This is currently broken 😬 I believe the points are hidden by other layers but currently they're not visible. Totally unrelated to this PR cc @Lebedinsky